api/WheelEvent.json

```
@@ -326,7 +326,8 @@
               "version_added": null
             },
             "safari": {
-              "version_added": false
+              "version_added": "15",
+              "notes": "Pinch-to-zoom gestures produce <code>WheelEvent</code>s in addition to non-standard <a href='https://developer.mozilla.org/docs/Web/API/GestureEvent'><code>GestureEvent</code></a>s. In prior versions, only <code>GestureEvent</code>s were emitted."
```

For long-term maintainability, I'm going to suggest we drop this note. The discussion of `WheelEvent` more or less recaps the structured data itself, while the discussion of the non-standard event support is shown through several specific features, such as `api.Element.gesturestart_event`. If support for the non-standard events ever changes, then this note is very likely to fall out of sync with those other features.

```suggestion
              "version_added": "15"
```